esri-mock-service/app.py

```diff
@@ -0,0 +1,164 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/findAddressCandidates")
+def findAddressCandidates():
+    jsonstr = """{
```

Comment:
Do you think it would be more maintainable if we stored these responses as json files, then read them into memory at server start?

Reply:
We do have these stored in `test/resources/esri` used by the unit tests, so hopefully we can use the same ones.